### PR TITLE
MDCT-2393: PDF Margins Issue

### PR DESCRIPTION
**Pull request by karla-vm**
### Description

When an MCR user would _IRL_ 🖨️ print 🖨️ a PDF report, oftentimes their top and bottom margins would cut off on the _IRL_ 📃 paper 📃 

### Related ticket(s)

MDCT-2393

---
### How to test

- Create a report in MCR
- Review PDF
- Download PDF

... I'm not going to ask you to _IRL_ print this, but if you're so inclined, you are welcome to 💯 

### Important updates

N/A

---
### Author checklist

- [ ] I have performed a self-review of my code
- [ ] I have added [thorough](https://bit.ly/3zPrxuZ) tests, if necessary
- [ ] I have updated relevant documentation, if necessary
---

_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_

### CODE REVIEW

**Review by ajaitasaini — DISMISSED** [review]
lgtm 😃

File: services/ui-src/src/components/export/ExportedSectionHeading.tsx [hunk 1]
```diff
@@ -60,5 +65,9 @@ const sx = {
   },
   spreadsheet: {
     margin: "1.5rem 0",
+    "@media print": {
+      // pageBreakBefore: "avoid",
```
Comment (gmrabian) [inline services/ui-src/src/components/export/ExportedSectionHeading.tsx:69]: this comment worth keeping?

**Review by gmrabian — DISMISSED** [review]
lgtm, did not test

**Review by cassandradanger — APPROVED** [review]
lgtm!

File: services/ui-src/src/components/export/ExportedSectionHeading.tsx (hunk 1 above)
Comment (karla-vm) [reply services/ui-src/src/components/export/ExportedSectionHeading.tsx]: removed :D

**Pull request merged**